lib/lita/handlers/active_random.rb

```diff
@@ -31,6 +32,10 @@ class ActiveRandom < Handler
       route /\bdo (yo)?u( even)? (work( )?out|lift)\b/i,     :do_u_workout,       command: true
       route /\Aping\z/i,                                     :ping,               command: true
 
+      def big_data response
+        response.reply 'https://s-media-cache-ak0.pinimg.com/736x/91/fd/52/91fd522284b34b4e133656f5c222c57f.jpg'
```

Comment:
Can you replace the link with this one? http://i.imgur.com/U6m4s4o.jpg

I like to host the images on imgur as much as possible so they don't get deleted by someone else.